### PR TITLE
Change PrintTime() to mimick original KSP behaviour

### PR DESCRIPTION
**Pull request by simonmeulenbeek**
Fix of issue #12 
PrintTime will give `valuesOfInterest`-amount of values from the first non-zero value. Returns highest significance first (years). 
```
PrintTime("0y 0d 0h 0m 0s", 0, false) = "0s"
PrintTime("0y 1d 2h 3m 4s", 0, false) = "1d"
PrintTime("0y 0d 2h 3m 4s", 0, false) = "2h"
PrintTime("0y 0d 2h 3m 4s", 3, false) = "2h, 3m, 4s"
PrintTime("0y 1d 2h 3m 4s", 1, false) = "1d, 2h"
PrintTime("1y 0d 0h 3m 4s", 1, false) = "1y, 0d"
PrintTime("1y 0d 0h 3m 4s", 2, false) = "1y, 0d, 0h"
```

**Note**:
The count of returned-values will not be the same as valuesOfInterest.
If the time is 0y 0d 0h 0m 1s it will only ever output one value ('1s') even if valuesOfInterest is higher.

This should mimick the original KSP code, but is pretty confusing so I'd thought I'd make a note of it.

### CODE REVIEW

**Comment by simonmeulenbeek** [comment]
**Note:** 
The count of returned-values will not be the same as `valuesOfInterest`. 
If the time is `0y 0d 0h 0m 1s` it will only ever output one value ('1s') even if `valuesOfInterest` is higher.

This should mimick the original KSP code, but is pretty confusing so I'd thought I'd make a note of it.

**Comment by jwvanderbeck** [comment]
Yeah that original code is confusing as all hell

File: RSSTimeFormatter/RealDateTimeFormatter.cs [hunk 1]
```diff
@@ -65,30 +65,31 @@ public string PrintTimeStampCompact(double time, bool days = false, bool years =
 		}
 		public string PrintTime(double time, int valuesOfInterest, bool explicitPositive)
 		{
-			// This is a downright strange and confusing method but as I understand it
```
Comment (jwvanderbeck) [inline RSSTimeFormatter/RealDateTimeFormatter.cs:68]: If the comment is erroneous, let's update the comment to be correct, but leave it in as this is such a confusing piece of code it helps to have the comment for future head-scratching.

Comment (simonmeulenbeek) [reply RSSTimeFormatter/RealDateTimeFormatter.cs:68]: Added explanation in comment

**Pull request closed without merging**